guides/availability-alerts.md

```diff
@@ -0,0 +1,57 @@
+# Availability Alert Setup Guide
+
+Availability alerts send web request to an application at regular intervals from various different Azure region around the world. It will alert a set Action Group if an application isn't responding, or if it doesn't respond within a given timescale.
+
+The simplest test is a URL ping. A URL ping relys on a public accessible URL.
+
+## How to add an availability alert
+
+1. Select an Application Insight resource

```

Comment:
I think this needs a line to say the following steps are using the Azure Portal.

Maybe a line to say whether the steps need to be run in each subscription where alerting is required.